ui/app/js/router.js

```diff
@@ -74,6 +74,12 @@ angular.module("app").config(function($routeProvider, $locationProvider) {
     reloadOnSearch: false
   })
 
+  .when('/rules/:id/:pr_ch_filter', {
```

Comment:
I don't think /rules/:id is what you want here - that's the history page, not the primary Rules page. Maybe we can use query args instead, to avoid the need for an additional route altogether, and get rid of the csv values? Eg: /rules?product=Firefox&channel=beta